Containerfile

```diff
@@ -1,23 +1,10 @@
-FROM golang:1.24 AS build-stage
-
-WORKDIR /build
-
-COPY go.mod go.sum ./
-RUN go mod download
-
-COPY . ./
-
-RUN --mount=type=cache,target=/root/.cache/go-build \
-    --mount=type=cache,target=/go/pkg \
-    CGO_ENABLED=0 GOOS=linux go build -o ./app cmd/app/main.go
-
 FROM gcr.io/distroless/static-debian12:nonroot
```

Comment:
We've discussed this at DM, but in my op - I should be able to build container without installing third-party tools like goreleaser